### PR TITLE
Add list connections endpoint

### PR DESCRIPTION
**Pull request by mthenw**

### CODE REVIEW

File: httpapi/httpapi.go [hunk 1]
```diff
@@ -17,11 +17,26 @@ type HTTPAPI struct {
...
+func (h HTTPAPI) listConnections(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+	w.Header().Set("Content-Type", "application/json")
+	encoder := json.NewEncoder(w)
+
+	space := params.ByName("space")
+	connections, err := h.Connections.ListConnections(space)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		encoder.Encode(&Response{Errors: []Error{{Message: err.Error()}}})
+	} else {
```
Comment (sebito91) [inline httpapi/httpapi.go]: Slight change, would outdent this here and add a `return` to the `if` block

**Review by sebito91 — CHANGES_REQUESTED** [review]
Small nit

File: httpapi/httpapi.go (hunk 1 above)
Comment (mthenw) [reply httpapi/httpapi.go]: @sebito91 done

**Review by sebito91 — APPROVED** [review]
LGTM

**Pull request merged**